proto/server.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
```

Comment:
Unfortunately not generated directly by go. We need to document building the protos, but in short you need to install protobuf3 beta 1, `go get -a github.com/golang/protobuf/protoc-gen-go`, `go get google.golang.org/grpc`, then you should be able to `make protos` (see the Makefile for the exact command that gets run)